Modules/Euler/Euler_UtilitiesModule_NonRelativistic.F90

```diff
@@ -90,9 +91,9 @@ SUBROUTINE ComputePrimitive_Scalar &
     ( N, S_1, S_2, S_3, G, Ne, D, V_1, V_2, V_3, E, De, &
       Gm_dd_11, Gm_dd_22, Gm_dd_33 )
 
-#if defined(THORNADO_OMP_OL)
+#if   defined( THORNADO_OMP_OL ) && !defined( THORNADO_EULER_NOGPU )
```

Comment:
We still need these routines on the GPU outside of just the Hydro runs, so don't add the `THORNADO_EULER_NOGPU` in here.

Reply:
Okay, I can remove those. I'll look around and see if that happened elsewhere